openshift/tests-extension/Makefile

```diff
@@ -144,7 +144,7 @@ clean-metadata: #HELP Remove 'codeLocations' from metadata JSON
 
 .PHONY: verify-metadata #HELP To verify that the metadata was properly update
```

Comment:
Could we have a target specific for the verify-bindata?
Or change it to verify and update the helpers